config/locales/en.yml

```diff
@@ -2853,6 +2853,21 @@ en:
     no_zip: This file is not a .zip archive.
     to_big: This file is too big.
   mailer:
+    warning: "MaMpf Warning"
+    pending_deletion_subject: "%{num_days_until_deletion} days before your
+      account will be deleted"
+    deletion_subject: "Account deleted"
+    pending_deletion_body_html: >
+      We've noticed that you haven't logged in to your MaMpf account in the last
+      6 months. Your account will be deleted in %{num_days_until_deletion} days
+      <strong>unless you log in to your MaMpf account before then</strong>.
+      Please note that once your account is deleted, all of your data stored
+      at MaMpf will be permanently lost.
+    deletion_body_html: >
+      <strong>Your MaMpf account has been deleted entirely as you have not
+      logged in for more than 6 months.</strong> Note that we've sent several
+      reminders beforehand. All of your data stored at MaMpf has been
+      permanently lost. Thanks for using MaMpf.
```

Comment:
Shouldn't it be "have been lost"?

Reply:
In my opinion,  `data` should be treated as mass noun since "The data show us that.." doesn't just sound right in my ears, whereas "The data show**s** us that..." sounds better. I'm backed by a [random translation on Deepl](https://www.deepl.com/de/translator#de/en/Alle%20deine%20Daten%20sind%20verloren%20gegangen.) and many answers on StackOverflow, e.g. [this one](https://english.stackexchange.com/a/73963) as well as [Grammer Monster](https://www.grammar-monster.com/lessons/data_singular_or_plural.htm)  (don't know how trustworthy the latter one is haha). At least, they have a summary I can live with:

> "Data" is the Latin plural of "datum," but "data" is transitioning to a singular word that means "information." Many academics and statisticians retain "data" as a plural word, but if you naturally treat it as singular, that is now fully acceptable.

Also [thesaurus](https://www.thesaurus.com/e/grammar/data-is-or-data-are/) (which is hopefully more trustworthy) speaks for my point of view:

> The word data can be either singular or plural depending on meaning and context. In general usage, data is treated as singular when used as a mass noun to mean “information” and as plural when used to mean “individual facts.” In scientific and academic writing, data is almost always used as a plural noun. In digital technology, data is usually treated as a singular mass noun to mean “digitally stored information.” 

Especially:

> In the context of computers and digital technology, data is usually treated as a singular mass noun to refer to digitally stored information. 
> For example:
> User data is gathered by many different social media companies. 
All of the data on the hard drive was corrupted by a computer virus.

Reply:
> In scientific and academic writing, data is almost always used as a plural noun

That's probably the reason why I view it as a plural noun by default. But if digital natives think of it as a singular noun, that's fine by me.